rspec-mode.el

```diff
@@ -635,6 +651,15 @@ file if it exists, or sensible defaults otherwise."
          (substring file (length remote))
        file))))
 
+(defun rspec--vagrant-wrapper (command)
+  (if (rspec-vagrant-p)
+      (format "vagrant ssh -c 'cd %s; %s'"
+              rspec-vagrant-cwd
+              (replace-regexp-in-string (rspec-project-root)
+                                        rspec-vagrant-cwd
```

Comment:
Actually, maybe this replacement would be better performed in `rspec--shell-quote-local` (that function can be renamed).